src/dune_pkg/substs.ml

```diff
@@ -0,0 +1,12 @@
+open Stdune
+
+let subst _env ~path ~target =
+  let env full_variable =
+    let _variable = OpamVariable.Full.variable full_variable in
+    None
```

Comment:
What is this about? Does the opam code work via side effects?

Reply:
No, this is just temporary code as I try to hook it up with the Env.